build/images/base-windows/Dockerfile

```diff
@@ -61,3 +61,32 @@ COPY --from=windows-utility-base ["C:\\\\wins","C:\\\\wins"]
 COPY --from=windows-golang ["C:\\\\Program Files\\\\Go", "C:\\\\Program Files\\\\Go"]
 
 RUN setx /m PATH "C:\Program Files\7-Zip;C:\git\bin;C:\git\usr\bin;C:\mingw64\bin;C:\Program Files\Go\bin;%GOPATH\bin%;%PATH%"
+
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
+ARG WIN_OVS_VERSION
+
+# Switch to a privileged user
+USER ContainerAdministrator
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+ENV WIN_OVS_VERSION ${WIN_OVS_VERSION}
+
+# Install VC
+RUN curl.exe -Lo C:\vc_redist.x64.exe https://aka.ms/vs/17/release/vc_redist.x64.exe; \
+    powershell -Command  "Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait"
+
+# Download and extract OVS
+RUN curl.exe -fLo C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip https://downloads.antrea.io/ovs/ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip;  \
+    powershell -Command Expand-Archive C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip -DestinationPath C:\  ;  \
+    rm C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip
+
+# Download and extract OpenSSL
+RUN curl.exe -Lo C:\openssl-1.0.2u-x64_86-win64.zip https://indy.fulgan.com/SSL/openssl-1.0.2u-x64_86-win64.zip; \
+    powershell -Command Expand-Archive C:\openssl-1.0.2u-x64_86-win64.zip -DestinationPath C:\  ;  \
+    rm C:\openssl-1.0.2u-x64_86-win64.zip
+
+RUN cp C:/libeay32.dll C:/openvswitch/usr/sbin/ ; \
```

Comment:
Using a seperate `Run` to move/copy the files make the files stored in multiple layers of the image, increasing the image size. It should be done in one `RUN`.

Reply:
Combining commands into a single RUN instruction can help reduce the number of layers in the Docker image and improve build performance. However it can impact readability and maintainability when doing so. If the commands become too long or complex, it might be better to split them into separate RUN instructions for better clarity.

Reply:
It's a very typical practice, downloading, extracting, installing a module, and deleting unneeded files is done in one RUN instruction. Sometimes even multiple modules may be merged into one RUN if they share the source cache. You may get the conclusion if you look at every existing dockerfile of antrea and most popular cloud native projects.

I don't get how splitting it into two RUN improves readability and maintainability. Commands should be connected via line breaker, there is one instruction per line anyway.

Reply:
> t's a very typical practice, downloading, extracting, installing a module, and deleting unneeded files is done in one RUN instruction.

Right, so line 79 looks fine separately , we can combine 84 and 88